lib/ansible/modules/database/postgresql/postgresql_privs.py

```diff
@@ -70,6 +70,11 @@
         for the implicitly defined PUBLIC group.
       - 'Alias: I(role)'
     required: yes
+  target_roles:
+    description:
+      - Comma-separated list of existing role (user/group) names to set as the
+        default permissions for database objects subsequently created by them.
+    version_added: 2.8
```

Comment:
Please, related you last changes, also change the parameter description "Comma-separated list of existing role (user/group) names to set as the default permissions for database objects subsequently created by them. I(target_roles) is only available with I(type=default_privs)."

Reply:
@Andersson007 thanks for your good hints. I forgot the "state=absent" part for this case. I added it and also added your suggestion for the description.

Reply:
Please add parameter types to every parameter.
In this case this would be better as:
```yaml
  target_roles:
    description:
    - A list of existing role (user/group) names to set as the
      default permissions for database objects subsequently created by them.
    - Parameter I(target_roles) is only available with C(type=default_privs).
    type: list
    version_added: '2.8'
```